File: Documentation/4.1/Samples/java/src/test/java/net/ravendb/ClientApi/Session/Querying/HowToFilterByField.java
public class HowToPerformProximitySearch {

    private interface IFoo<T> {
        //region whereexists_1
        IDocumentQuery<T> whereExists(String fieldName);
        //endregion
    }

    public void sample1(IDocumentSession session) {
        //region whereexists_2
        List<Employee> results = session
                .advanced()
                .documentQuery(Employee.class)
                .whereExists("FirstName")
                .toList();
        //endregion

        //region whereexists_3
        List<Employee> results = session
		        .advanced()
		        .documentQuery(Employee.class)
		        .whereExists("Address.Location.Latitude")
		        .toList();
        //endregion
    }

File: Documentation/5.4/Samples/nodejs/client-api/session/querying/howToFilterByField.js
import { DocumentStore } from "ravendb";

const documentStore = new DocumentStore();

class Syntax {
    //region whereExists_syntax
    whereExists(fieldName);
    //endregion
}

async function whereExists() {

    //region whereExists_1
    // Only documents that contain field 'firstName' will be returned
    
    await session.query({ collection: "Employees" })
        .whereExists("firstName");
        .all();
    //endregion

    //region whereExists_2
    // Only documents that contain the 'latitude' property in the specified path will be returned
    
    await session.query({ collection: "Employees" })
        .whereExists("address.location.latitude");
        .all();
    //endregion
}
